core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -252,6 +274,14 @@ protected IdGenerator getIdentityGenerator() {
     return idGenerator;
   }
 
+  public boolean isShutDownInProgress() {
+    return shutDown.get();
+  }
+
+  private void setShutDownInProgress() {
```

Comment:
Since this isn't a standard setter, how about, more explicit?

```suggestion
  private void enableShutdown() {
```